test/Microsoft.Sbom.Api.Tests/Config/ConfigFileParserTests.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Sbom.Common;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Microsoft.Sbom.Api.Config.Tests;
+
+[TestClass]
+public class ConfigFileParserTests
+{
+    private Mock<IFileSystemUtils> mockFileSystemUtils;
+    private ConfigFileParser testSubject;
+    private readonly string filePathStub = "test-path";
+    private readonly string contentStub = "{\"PackageSupplier\": \"TestSupplier\",\"BuildDropPath\": \"$(BuildDropPathEnvVar)\"}";
+    private readonly string envVarName = "BuildDropPathEnvVar";
+    private readonly string envVarValue = "TestPath";
+
+    [TestInitialize]
+    public void Initialize()
+    {
+        mockFileSystemUtils = new Mock<IFileSystemUtils>();
```

Comment:
I think this can be strict?
```suggestion
        mockFileSystemUtils = new Mock<IFileSystemUtils>(MockBehavior.Strict);
```